src/tests/ftest/tags.py

```diff
@@ -267,6 +287,62 @@ def __parse_avocado_tags(text):
         return set(','.join(tag_strings).split(','))
 
 
+class TestYamlData():
```

Comment:
Maybe `TestConfig`? "data" is very generic. Everything is data :)

Reply:
Sounds good to me.